openmetadata-service/src/main/java/org/openmetadata/service/resources/teams/UserResource.java

```diff
@@ -177,6 +163,12 @@ public void initialize(OpenMetadataApplicationConfig config) throws IOException
     SmtpSettings smtpSettings = config.getSmtpSettings();
     this.isEmailServiceEnabled = smtpSettings != null && smtpSettings.getEnableSmtpServer();
     this.loginAttemptCache = new LoginAttemptCache(config);
+    if (config.getAuthenticationConfiguration().getProvider().equals("ldap")) {
```

Comment:
can we do as part of OpenMetadataApplication to instantiate the implementation just like we do for DefaultAuthorizer?